app/models/claim.rb

```diff
@@ -202,23 +202,23 @@ def student_loan_country_with_one_plan?
   # Returns true if the claim has a confirmed identity. For now this can only
   # be done through GOV.UK Verify, but will change in the future.
   def identity_confirmed?
-    verified_fields.any?
+    govuk_verify_fields.any?
   end
 
   def name_verified?
-    verified_fields.include?("first_name")
+    govuk_verify_fields.include?("first_name")
   end
 
-  def address_verified?
-    (ADDRESS_ATTRIBUTES & verified_fields).any?
+  def address_from_govuk_verify?
```

Comment:
Wondering if this should be moved below all the `*_verified?` methods, just so it's clearer the intent is different